addons/common/backwards_comp.sqf

```diff
@@ -71,3 +71,11 @@ CBA_fnc_intToString = {
     if (isNil "_int") exitWith {""};
     str _int
 };
+
+CBA_fnc_inArea = {
```

Comment:
Indeed, so we should not add to the problem.

Instead, change the old function file and keep its CfgFunctions entry.